superset/tags/schemas.py

```diff
@@ -59,22 +59,24 @@ class TagPostSchema(Schema):
     description = fields.String(required=False, allow_none=True)
     # resource id's to tag with tag
     objects_to_tag = fields.List(
-        fields.Tuple((fields.String(), fields.Int())), required=False
+        fields.Tuple((fields.String(), fields.Int())), required=False, allow_none=True
```

Comment:
Why do we allow `None` here? It seems to be it's better to not allow `None`, and filter them out in the frontend, since it doesn't make sense for the client to send a null value here.